bigchaindb/tendermint/lib.py

```diff
@@ -152,6 +152,18 @@ def get_block(self, block_id, include_status=False):
         else:
             return block
 
+    def get_blocks_status_containing_tx(self, txid):
+        """Retrieve block id containing transaction id `txid`
```

Comment:
Maybe:

Retrieve the list of blocks (block ids) containing a transaction with transaction id `txid`

Reply:
:+1: